lightning_pose/utils/scripts.py

```diff
@@ -84,6 +85,15 @@ def get_dataset(
             do_context=cfg.model.model_type == "heatmap_mhcrnn" or cfg.model.do_context,
             uniform_heatmaps=cfg.training.get("uniform_heatmaps_for_nan_keypoints", False),
         )
+    elif cfg.model.model_type == "MultiviewHeatmapDataset":
```

Comment:
should move this inside the above `elif` statement, so that it looks like this:
```
elif cfg.model.model_type == "heatmap" or cfg.model.model_type == "heatmap_mhcrnn":
    if cfg.data.get(“view_names”, None) and len(cfg.data.view_names) > 1:
        dataset = MultiviewHeatmapDataset(...)
    else:
        dataset = HeatmapDataset(...)
```

This allows the `cfg.model.model_type` field to continue representing the model type irrespective of whether or not we do multiview; we might want a simple multiview heatmap model, or a more complex multiview heatmap_mhcrnn model.

Reply:
for now we are not supporting multiview operations with the regression model, so we should also do:
```
if cfg.model.model_type == "regression":
    if cfg.data.get(“view_names”, None) and len(cfg.data.view_names) > 1:
        raise NotImplementedError("Multi-view support only available for heatmap-based models")
    else:
        dataset = BaseTrackingDataset(...)
```